transforms/underscore-to-lodash-native.js

```diff
@@ -0,0 +1,184 @@
+const DEFAULT_OPTIONS = {
+  'split-imports': false,
+};
+
+const NATIVE_METHODS = {
+  forEach: 'forEach',
+  each: 'forEach',
+  map: 'map',
+  collect: 'map',
+  filter: 'filter',
+  select: 'filter',
+  every: 'every',
+  some: 'some',
+  find: 'find',
+  detect: 'find',
+  contains: 'includes',
+  reduce: 'reduce',
+  inject: 'reduce',
+  indexOf: 'indexOf',
+  lastIndexOf: 'lastIndexOf',
+  first: (j, identifier) => j.memberExpression(identifier, j.literal(0)),
+  last: (j, identifier) => j.memberExpression(identifier,
+    j.binaryExpression('-',
+      j.memberExpression(identifier, j.identifier('length')),
+      j.literal(1)
+    )
+  ),
+};
+
```

Comment:
Just a bit of ⚠️ : Methods like `_.each`, `_.map`, `_.filter`, `_.every`, `_.some`, `_.includes` aren't 💯 equiv to built-ins. They're "collections" methods which means they work with array-likes and objects. That and iteratee shorthands are some of the reasons to use them.